Automattic/jetpack
# Boost: Update query params

## Proposed changes:

* Whitelist `jb-disable-modules` for SEO plugins.
* Some small refactor.

### Other information:


## Jetpack product discussion
p1711350424667999/1711195573.320329-slack-C016BBAFHHS

## Does this pull request change what data or activity we track or use?
No

## Testing instructions:
Test Cloud CSS and local Critical CSS with Yoast and All in One SEO plugin and enable the query parameter removing features. Make sure both C.CSS generation work.
